hub/powertoys/grouppolicy.md

```diff
@@ -23,48 +27,161 @@ Since version 0.64, PowerToys is released on GitHub with [Administrative Templat
 2. Copy the "PowerToys.admx" file to the PolicyDefinition folder. (Example: %systemroot%\sysvol\domain\policies\PolicyDefinitions)
 3. Copy the "PowerToys.adml" file to the matching language folder in the PolicyDefinition folder. Create the folder if it doesn't already exist. (Example: %systemroot%\sysvol\domain\policies\PolicyDefinitions\EN-US)
 4. If your domain has more than one domain controller, the new ADMX files will be replicated to them at the next domain replication interval.
+### Import the administrative template in Intune
+
+You can find all instructions on how to import the tdministrative templates in Intune [here](https://learn.microsoft.com/mem/intune/configuration/administrative-templates-import-custom#add-the-admx-and-adml-files).
```

Comment:
```suggestion
You can find all instructions on how to import the administrative templates in Intune [here](/mem/intune/configuration/administrative-templates-import-custom#add-the-admx-and-adml-files).
```
Fixing build suggestion and spelling/typo

Reply:
Thank you. Wasn't sure how to fix it correctly.